frontend/src/app/main/ui/workspace/right_header.cljs

```diff
@@ -194,7 +198,15 @@
                        (dw/clear-edition-mode)))
 
            (st/emit! (-> (dw/toggle-layout-flag :document-history)
-                         (vary-meta assoc ::ev/origin "workspace-header")))))]
+                         (vary-meta assoc ::ev/origin "workspace-header")))))
+
+        open-share-dialog
+        (mf/use-fn
+         (fn []
+           (mf/deps team)
```

Comment:
This deps are ignored, you need to pass deps as first parameter to mf/use-fn

Reply:
Ops! Fixed